GP05MOAS-GL363/GP05MOAS-GL363_R00006_ingest.csv

```diff
@@ -1,5 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00006/merged/gp_*.mrg,GP05MOAS-GL363-00-ENG000000,recovered_host,Available,
-mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00006/merged/gp_*.mrg,GP05MOAS-GL363-01-FLORDM000,recovered_host,Available,
+#mi.dataset.driver.moas.gl.flord_m.flord_m_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00006/merged/gp_*.mrg,GP05MOAS-GL363-01-FLORDM000,recovered_host,Available,A 3-wavelength fluorometer was deployed (instead of 2-wavelength). Parser in the next line.
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/R00006/merged/gp_*.mrg,GP05MOAS-GL363-01-FLORDM000,recovered_host,Available,A 3-wavelength fluorometer (FLORTM) was deployed in place of a 2-wavelength (FLORDM). This is the parser for 3-wavelength fluorometer
```

Comment:
looks good - matches up with how we handled it on GP05MOAS-GL276 R00002 ingestion csv